playground/backend/cmd/server/controller_test.go

```diff
@@ -80,6 +86,12 @@ func setup() *grpc.Server {
 	// setup cache
 	cacheService = local.New(context.Background())
 
+	// setup entity storage
+	snippetDb, err = localdb.New()
+	if err != nil {
+		panic(err)
```

Comment:
please use panic only if it is inevitable, just double check that we cannot replace it with other options
https://github.com/uber-go/guide/blob/master/style.md#dont-panic